libs/data-access/database/src/lib/learningDiary/learningDiary.ts

```diff
@@ -0,0 +1,284 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+enum LearningProgress {
+	NOT_STARTED = "NOT_STARTED",
+	STARTED = "STARTED",
+	FINISHED = "FINISHED"
+}
```

Comment:
enum existiert bereits in prisma schmema. weg finden zum wiederverwenden

Reply:
ist enfernt